### PR TITLE
Editor for Woocommerce transactional emails [MAILPOET-2281]

### PR DESCRIPTION
**Pull request by amine-mp**
[MAILPOET-2281]

[MAILPOET-2281]: https://mailpoet.atlassian.net/browse/MAILPOET-2281

### CODE REVIEW

File: assets/js/src/newsletter_editor/components/sidebar.js [hunk 1]
```diff
@@ -77,6 +77,11 @@ SidebarView = Marionette.View.extend({
       }
     },
   },
+  templateContext: function () { // eslint-disable-line func-names
```
Comment (NeosinneR) [inline assets/js/src/newsletter_editor/components/sidebar.js]: Why the eslint comment? It's disabled for the whole file on the first line

**Comment by codemonkey-jack** [comment]
This is now in product QA with Quentin.

**Comment by amine-mp** [comment]
@NeosinneR I just added a small commit to fix the border radius issue.

**Comment by codemonkey-jack** [comment]
Back with Quentin in product QA.

**Pull request merged**